modules/gdscript/tests/scripts/runtime/errors/op_by_zero/division_by_zero_int.gd

```diff
@@ -0,0 +1,3 @@
+func test():
+	var integer = 1
+	integer /= 0
```

Comment:
I'm not sure if validated evaluator is used in this case or not. Maybe yes, since it's a local variable and the inferred weak type doesn't change. But if not, we can't tell from the positive test result, since validated evaluators are not used for untyped variables. So I'd prefer to explicitly use typed variables.

```suggestion
	var integer := 1
	integer /= 0
```

or

```suggestion
	var integer: int = 1
	integer /= 0
```